programs/static/css/style.css

```
@@ -0,0 +1,4 @@
+.selector select {
+    resize: vertical;
+    overflow: auto;
```

This allows for both vertical and horizontal resize:
```suggestion
    resize: both;
    overflow: auto; 
    white-space: nowrap; 
    width: 100%;
    max-width: 100%; 
```